templates/tweet/login.html

```diff
@@ -0,0 +1,43 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Twitter</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="{% static 'css/main.css' %}" >
+    </head>
+    <body>
+        <div class="container">
+            <div class="row">
+                <div class="col-md-6">
+                    <img src="{% static 'image/twitter_logo.png' %}" alt="Error" width="350" height="268">
+                </div>
+                <div class="col-md-6">
+					<h1>Login</h1>
+					<h4>Already have an account.</h4>
+                    <form method="post" action="{% url 'user:Signin' %}">
+						{% csrf_token %}
+						<div>
+							<label>Email</label>
+							<input type="email"  name="email" required>
+						</div>
+						<div>
+							<label>Password</label>
+							<input type="password"  name="password" required>
+						</div>
+						<div>
+							<input type="submit" value="Login">
+						</div>
+						<a href="{% url 'user:Signup' %}">Register</a>
+					</form>
+                </div>
```

Comment:
index properly. See https://www.geeksforgeeks.org/top-vs-code-extensions-for-python-django/

Reply:
![Screenshot from 2024-05-08 11-38-22](https://github.com/GaganDureja/twitter/assets/53633318/12fb641d-d9f2-4900-97ea-9f20f8ce565f)


This is properly done. Although github is showing like this

Reply:
1. See the difference between tabs vs spaces.
2. Did you install extensions as shared in the link above ?

Enable settings in VSCODE 

```
  "editor.renderWhitespace": "all",
  "files.trimTrailingWhitespace": true,
  "editor.indentSize": "tabSize",
  "editor.tabSize": 2,
```

<img width="1195" alt="Screenshot 2024-05-13 at 8 55 46 AM" src="https://github.com/GaganDureja/twitter/assets/845708/4654de63-4565-4e1d-8fc7-0b659e5c1544">